src/core/server/integration_tests/saved_objects/migrations/group3/incompatible_cluster_routing_allocation.test.ts

```diff
@@ -34,12 +34,7 @@ const { startES } = createTestServers({
   settings: {
     es: {
       license: 'basic',
-      dataArchive: Path.join(
-        __dirname,
-        '..',
-        'archives',
-        '8.0.0_v1_migrations_sample_data_saved_objects.zip'
```

Comment:
Can this archive be removed?

Reply:
I had high hopes that we'll be able to reuse it eventually. But happy to remove it if you think it's best to recover it when needed.